docs/userguide/Networking/USERGUIDE.md

```diff
@@ -468,3 +482,136 @@ $port->delete();
 ```
 
 [ [Get the executable PHP script for this example](/samples/Networking/delete-port.php) ]
+
+## Security Groups
+
+A security group is a named container for [security group rules](#security-group-rules).
+
+### Create a security group
+
+This operation takes one parameter, an associative array, with the following keys:
+
+| Name | Description | Data type | Required? | Default value | Example value |
+| ---- | ----------- | --------- | --------- | ------------- | ------------- |
+| `name` | A human-readable name for the security group. This name might not be unique. | String | Yes | - | `new-webservers` |
+| `description` | Description of the security group. | String | No | `null` | `security group for webservers` |
+
+You can create a security group as shown in the following example:
+
+```php
+$securityGroup = $networkingService->createSecurityGroup(array(
+    'name' => 'new-webservers',
+    'description' => 'security group for webservers'
+));
+/** @var $securityGroup OpenCloud\Networking\Resource\SecurityGroup **/
+```
+
+[ [Get the executable PHP script for this example](/samples/Networking/create-security-group.php) ]
+
+### List security groups
+
+You can list all the security groups to which you have access as shown in the following
+example:
+
+```php
+$securityGroups = $networkingService->listSecurityGroups();
+foreach ($securityGroups as $securityGroup) {
+    /** @var $securityGroup OpenCloud\Networking\Resource\SecurityGroup **/
+}
+```
+
+[ [Get the executable PHP script for this example](/samples/Networking/list-security-groups.php) ]
+
+### Get a security group
+
+You can retrieve a specific security group by using that security group's ID, as shown in the
+following example:
+
+```php
+$securityGroup = $networkingService->getSecurityGroup('2076db17-a522-4506-91de-c6dd8e837028');
```

Comment:
I checked the other User Guides and they are using literals instead of placeholders for IDs. So, for consistency sake, I will leave this as-is.